typescript/package.json

```diff
@@ -1,49 +1,27 @@
 {
-  "name": "@coinbase/cdp-sdk",
-  "version": "1.1.2",
-  "description": "CDP SDK",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "repository": "https://github.com/coinbase/cdp-sdk",
+  "private": true,
+  "packageManager": "pnpm@10.5.2",
+  "type": "module",
```

Comment:
This tells Node that we are writing our code in the ESM format, which now [enforces mandatory file extensions](https://nodejs.org/api/esm.html#mandatory-file-extensions), which is why we end up having to change all of our import paths to end in `.js`